### DISCUSSION TITLE
Can FDS accurately predict backlayering/critical velocity in tunnels?

### DISCUSSION THREAD

**Discussion by ThijsH24** [firemodels/fds#13631] [Q&A]
I’m simulating a fire of 100 MW inside a tunnel with dimensions (L: 300m, W: 15.2m, H: 6m). The tunnel has an open boundary at one end and a fixed velocity at the inlet. In the FDS validation guide, there are comparisons made with similar experiments, such as the Memorial Tunnel test. However, these graphs show quite some differences, and no definitive conclusions are made within the validation guide. By looking at the graphs, I would say that FDS does not accurately predict backlayering.
Does anyone have an opinion about this? Can FDS, for example, be used in the design of real-sized tunnels, or is it not accurate enough? Or do we simply not know yet? How should I interpret the data I get from my simulations? I’m sorry if this sounds a bit vague, but I’m just interested in whether someone has an opinion about simulating real-sized tunnel fires with FDS.

**Comment by mcgratta** [comment]
As a rule, we don't draw conclusions in the FDS Validation Guide as to whether the model is "accurate enough" because there are no such performance criteria that are universally accepted. We simply present the results along with some simple statistics that quantify model bias and scatter. We also try as much as possible to simulate all the experiments for a given test series and present the results, good or bad. The rest is up to you to decide if the model is appropriate or not for your application. 

If you believe that FDS is not accurately predicting backlayering, I invite you to look at the half-dozen or so empirical correlations in the literature or run another CFD model. But whatever you do, look at all 100 Memorial Tunnel experiments and see how well your alternative method performs. If you are not happy with any, then you'll just have to run your own set of full-scale experiments.

**Reply by ThijsH24** [reply]
The question is more about how to justify the obtained results, assuming FDS is the best tool available at the moment. It is understandable that the simulated data does not fully correspond with the Memorial Tunnel test data. Besides numerical errors, there are also inaccuracies in the input parameters used and errors in the experimental data itself. But then, what foundational basis can you refer to? What are the limitations of FDS in determining back layering and critical velocity? Where should caution be applied?

Ideally, I would like to determine an estimated error margin for the critical velocity obtained by using FDS. Do you have any suggestions for how to approach this?

**Reply by mcgratta** [reply]
With regard to the Memorial Tunnel tests, you cannot assess the accuracy of the input parameters. We used the "nominal" values of everything -- fan flow rates, heat release rates, etc. There was lots of equipment in the tunnel, stuff hanging on the walls. At best, you get a rough sense of whether the model over or under-predicts the measured backlayer distance by, say, up to 30% or whatever is listed in the tables. 

The question to ask is -- in designing a smoke exhaust system, how accurate do you need to be given that there is going to be some sort of safety margin and/or you are designing for the 98th percentile fire size. I've always thought that the choice of an extreme fire scenario to design against is really how the safety factor is applied.

**Reply by ThijsH24** [reply]
Yes, you're right. You're testing for the worst-case scenarios. Therefore, you don't have to be extremely accurate; it won't come down to the last 0.1 m/s for the critical velocity. Still, when you present results, you need to be able to justify them. The argument—"we don't know the accuracy, but that doesn't matter because we don't have to be accurate since we already include a safety margin"—is not very strong. Not having to be super accurate is independent of how accurate the results are.

So, for example, I simulate a 100 MW fire inside a tunnel. The results show that there is no backlayering. Now I present the results to someone else. This person asks, "How accurate are these results? How sure are you that in reality, there is also no backlayering?" In that case, out of all the possible arguments, what do you think is the strongest one?

**Reply by mcgratta** [reply]
The Validation Guide presents the results of the tunnel simulations and tabulates the degree to which these results differ either from the experiments or the correlations derived from those experiments. What I am saying is that it is not possible to quantify the accuracy of some of the model input parameters because they are simply the nominal values (like fan speed) presented in the experimental report. You can only express the degree to which the model and experimental measurement of, say, the backlayering length, agree or disagree. If you want to know the MODEL uncertainty, you first need to assess the EXPERIMENTAL uncertainty. 

In your case, you would have to assume an uncertainty of, say, the HRR, and, say, the volume flow of the fans. Then you can start to put bounds around the backlayer distance. If the Validation Guide indicates that FDS tends to over-predict or under-predict the BL distance, you can factor that in too. Then you can look at the various empirical correlations and report what they predict. At the end of all this, you would present some kind of range of outcomes, but it won't be perfect because you do not have all the uncertainties in the experimental data. Welcome to  fire research.

**Reply by johodges** [reply]
It is important to be clear about the specific question(s) you are trying to answer with the model. Our team frequently uses FDS in support of design for tunnel applications; however, it is often in combination with other tools such as nodal models / hand calculations. A good sanity check is to compare the critical velocity that you are seeing in the CFD model with the modified Kennedy equation in the latest edition of NFPA 502. It is generally conservative compared with the memorial tunnel cases. I would be concerned with a model input or parameter if the required velocity upstream you are seeing with a model is significantly lower than the hand calculation. The cases where CFD is particularly useful in these scenarios tend to have complex cross sections, obstructions, or connecting flow paths which increase your pressure losses and have inhomogeneous downstream flows.

**Comment by ThijsH24** [comment]
Alright, got it. It’s an interesting topic. You are working with a lot of uncertainties; however, you have to do something. So, it’s a matter of doing a lot at once: numerics, empirical relations, guidelines, and hoping they all point in the same direction. From that, you have to conclude something without knowing precisely how accurate you are. 
Thank you for your insights, it helps a lot.

**Comment by ThijsH24** [comment]
One last question. What is the reason why the Memorial tunnel test is not listed in Table 3.48 and 3.49? Is it because the grid is not completely converged yet?

**Reply by mcgratta** [reply]
I just forgot to add it.
